src/components/Navbar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './Navbar.css';
+
+const Navbar = () => {
+  const linkStyle = {
+    margin: '1rem',
+    textDecoration: 'none',
+    color: 'orange',
+  };
+  return (
+    <>
+      <ul className="header">
+        <li>
+          <NavLink style={linkStyle} to="/">
+            Home
+          </NavLink>
+        </li>
+        <li>
+          <NavLink style={linkStyle} to="/calculator">
```

Comment:
- [ ] In App component For the calculator page you have defined the route path as `'/design'`, but here you are trying to access route `'/calculator'` and this path does not exist, and a screen shows an empty page with just a navbar. Please fix this issue and make sure that you use the correct path to goto defined route paths.
- [ ] 
## See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/172462999-31080f07-622a-4536-8a2a-c9e2987bcb08.png)